terraform/state_v1_test.go

```diff
@@ -1,118 +1,147 @@
 package terraform
 
 import (
+	"bufio"
 	"bytes"
-	"encoding/gob"
-	"errors"
-	"io"
 	"reflect"
-	"sync"
+	"strings"
 	"testing"
 
-	"github.com/mitchellh/hashstructure"
+	"github.com/davecgh/go-spew/spew"
```

Comment:
This isn't stray - we use it to output usable test expected/actual in the case of failure (e.g. L96)